envs/Loyalty 2 UAT/static-resources/salesforce/Records/ApexClass/FeatureFlagProvider.cls

```diff
@@ -13,6 +13,10 @@ public with sharing class FeatureFlagProvider implements IFeatureFlagProvider {
         return FeatureFlag__mdt.getAll();
     }
     
-  
+    public static void badQuery(){
+        for(Integer x=0;x < 10; x++){
+            List<Account> accts = [SELECT Id FROM Account];
+        }
```

Comment:
<!--sfdx-scanner-->
| Attribute | Value |
| --- | --- |
| Engine | pmd|
| Category | Performance |
| Rule | OperationWithLimitsInLoop |
| Severity | 3 |
| Type | Error |
| Message| [Avoid operations in loops that may hit governor limits](https://pmd.github.io/pmd-6.55.0/pmd_rules_apex_performance.html#operationwithlimitsinloop) |
| File | [envs/Loyalty 2 UAT/static-resources/salesforce/Records/ApexClass/FeatureFlagProvider.cls](../tree/2af81e0790fb6a5952df8918ad817a8f447fcb60/envs/Loyalty 2 UAT/static-resources/salesforce/Records/ApexClass/FeatureFlagProvider.cls) |